js/modules/sliderOurVisitors.js

```diff
@@ -0,0 +1,81 @@
+const slidesContainer = document.getElementById("ourVisitors__photo_container");
+const slidesQuantity = slidesContainer.querySelectorAll(".ourVisitors__photo").length;
+const SLIDER_MAX_POSSIBLE_POSITION = 0;
+let sliderMinPossiblePosition = -(slidesQuantity - 1)*100;
+let sliderCurrentShift = 0;
+let sliderCurrentPosition;
+
+const sliderRangeWidth = document.getElementById("slider_range").offsetWidth;
+const sliderThumb = document.getElementById("slider_thumb");
+const sliderThumbWidth = sliderRangeWidth/slidesQuantity;
+const SLIDER_THUMB_MIN_POSSIBLE_POSITION = 0;
+const sliderThumbMaxPossiblePosition = (slidesQuantity - 1)*100;
+let sliderThumbCurrentShift = 0;
+
+const btnPrevOurVisitors = document.getElementById("btn_prev_ourVisitors");
+const btnNextOurVisitors = document.getElementById("btn_next_ourVisitors");
+
+sliderThumb.style.width = sliderThumbWidth + "px"; 
+
+btnPrevOurVisitors.addEventListener("click", function(){
+	movePrevSlide();
+	moveThumbPrev();
+	if(btnNextOurVisitors.disabled){
+		enableButton(btnNextOurVisitors);
+	}
+});
+
+btnNextOurVisitors.addEventListener("click", function(){
+	moveNextSlide();
+	moveThumbNext();
+
+	if(btnPrevOurVisitors.disabled){
+		enableButton(btnPrevOurVisitors);
+	}
+});
+
+function movePrevSlide(){
+	sliderCurrentShift += 100;
```

Comment:
magic number as well as in other places in this document

Reply:
Done.